### PR TITLE
Config key

### PR DESCRIPTION
**Pull request by Savallator**
Keys are loaded from the config now.

### CODE REVIEW

File: EnoDatabase/EnoDatabaseUtils.cs [hunk 1]
```diff
@@ -26,8 +26,8 @@ public class EnoDatabaseUtils
     {
 
         const int DATABASE_RETRIES = 500;
-        public static readonly byte[] FLAG_SIGNING_KEY = Encoding.ASCII.GetBytes("suchasecretstornkkeytheywillneverguess");
-        internal static readonly byte[] NOISE_SIGNING_KEY = Encoding.ASCII.GetBytes("anotherstrenksecrettheyvref24tr");
+        //public static readonly byte[] FLAG_SIGNING_KEY = Encoding.ASCII.GetBytes("suchasecretstornkkeytheywillneverguess");
```
Comment (Trolldemorted) [inline EnoDatabase/EnoDatabaseUtils.cs:29]: We should remove this then :)

File: ctf.sample.json [hunk 2]
```diff
@@ -4,6 +4,8 @@
     "RoundLengthInSeconds": 60,
     "DnsSuffix": "eno.host",
     "TeamSubnetBytesLength": 6,
+	"FlagSigningKey": "ir7PRm0SzqzA0lmFyBfUv68E6Yb7cjbJDp6dummqwr0Od70Sar7P27HVY6oc8PuW",
```
Comment (Trolldemorted) [inline ctf.sample.json:7]: The indentation looks unintended

File: EnoEngine/FlagSubmissionEndpoint.cs [hunk 3]
```diff
@@ -116,7 +116,7 @@ async Task ReadPipeAsync(PipeReader reader, ChannelWriter<Task<FlagSubmissionRes
                     // Process the line.
                     if (teamId is long _teamId)
                     {
-                        var flag = Flag.Parse(line, EnoDatabaseUtils.FLAG_SIGNING_KEY, Logger);
+                        var flag = Flag.Parse(line, Encoding.ASCII.GetBytes(config.FlagSigningKey), Logger);
```
Comment (Trolldemorted) [inline EnoEngine/FlagSubmissionEndpoint.cs:119]: I think we should have a second config class which contains the key as bytes, and pass that around via DI, but that can go in another PR at a later time

**Comment by Savallator** [comment]
Yeah lol i used tabs in the ctf.json, but in reality there  were 4 spaces... in my editor it looked the same^^

**Pull request merged**